Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -237,6 +251,35 @@
   categories:
   - UplinkWeaponry
 
+
+- type: listing
+  id: WSPR-uplink-syndicate
+  name: WSPR
+  description: Новая винтовка синдиката, специально для ваших боёв
+  icon: { sprite: /Textures/_Goobstation/Objects/Weapons/Guns/Rifles/wspr.rsi, state: icon }
+  productEntity:  ClothingBackpackDuffelSyndicateFilledWSPR
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)